### DISCUSSION TITLE
Firmware

### DISCUSSION THREAD

**Discussion by RychuZyrafa** [deividAlfa/stm32_soldering_iron_controller#54] [General]
Hi,
I don't have T12 yet, but it's being delivered. Since there is no info about it, what are the benefits of installing this custom firmware? What does it do etc.

**Comment by deividAlfa** [comment]
Better ask this at the forum!
https://www.eevblog.com/forum/reviews/stm32-oled-digital-soldering-station-for-t12-handle/
